### ISSUE TITLE
Lint for common PII column names?

### ISSUE THREAD

**Issue by olirice** [supabase/splinter#25]
# Bug report
Any thoughts on how we could scan for problematic data types without a high false positive rate would be welcome

**Comment by olirice** [comment]
From Slack
```
what if we do a check where a table has columns like first_name, etc and has a rls policy with select true? that would have caught the case in the <example> table and i assume a lot of users would also add such a policy to silence warnings to test and forget to remove it when they are in prod
```

**Comment by olirice** [comment]
@bymayanksingh could you post something in this issue. GH won't let me assign it to you unless your a participant

**Comment by bymayanksingh** [comment]
Sure @olirice

**Comment by olirice** [comment]
Closed since this is less applicable now that we ERROR on insecure APIs
